core/src/test/java/org/apache/iceberg/rest/responses/TestLoadTableResponse.java

```diff
@@ -147,6 +147,23 @@ public void testRoundTripSerdeWithV2TableMetadata() throws Exception {
     assertRoundTripSerializesEquallyFrom(json, resp);
   }
 
+  @Test
+  public void testRoundTripSerdeWithV3TableMetadata() throws Exception {
+    String tableMetadataJson = readTableMetadataInputFile("TableMetadataV3ValidMinimal.json");
+    TableMetadata v3Metadata =
+        TableMetadataParser.fromJson(TEST_METADATA_LOCATION, tableMetadataJson);
+    // Convert the TableMetadata JSON from the file to an object and then back to JSON so that
+    // missing fields
```

Comment:
nit: this line could be merged with the one below

Reply:
Yeah, this was borrowed from above and looks like an auto-formatter issue.  Fixed.